Makefile.helper.mk

```diff
@@ -27,17 +27,21 @@ kube-lint: kube-linter
 lint: patch golangci-lint
 	$(GOLANGCILINT) run -v --timeout 5m0s
 
-verify: patch fmt vet
+verify: patch vet
```

Comment:
How do you feel about removing the `fmt` target entirely?

Reply:
I feel great :)

It was replaced by `gofmt -l` (inside the target) which will only check if there are diff without actually changing the files, it will also help us keep a clean `git diff`.

The reason is that in verify (that is currently added to prow), we don't need to format anything, just to check, if there are diff we will ask from the user to run `make fmt` but this shouldn't be a part of the unit-tests.

Reply:
Ohh, you meant remove `make fmt` entirely? 

I think we need it, if the `verify` target fails on CI, this is the easiest solution for the user to fix it.
Also, as @pacevedom said, we don't want to relay on each dev env formatter, but have a standard one for the project.

Reply:
Can we remove the `fmt` dependency from `manager` and `run` targets? I am happy with leaving it as a separate target so as to not force each dev IDE to have specific config, but maybe we could turn it into a standalone target.

Reply:
If you see a benefit in having `make fmt`, then let's leave it. But how about we remove it as a dependency of any other target, so it's never triggered in the CI? It serves no purpose there.

Reply:
REMOVED.